clocking/api/view.py

```diff
@@ -167,15 +123,15 @@ def get_persons(self, days):
         date1 = datetime(datetime.now().year, datetime.now().month, datetime.now().day)
         date2 = date1 + timedelta(hours=23, minutes=59, seconds=59)
 
-        if days == 'Astăzi':
+        if days == 'Astazi':
             return Person.query.filter(
                 Person.startdate.between(date1, date2)).order_by(Person.startdate)
-        elif days == 'Săptămâna curentă':
+        elif days == 'Saptamana curenta':
             while date1.weekday() != 0:
                 date1 -= timedelta(days=1)
             return Person.query.filter(
                 Person.startdate.between(date1, date2)).order_by(Person.startdate)
-        elif days == 'Luna curentă':
+        elif days == 'Luna curenta':
```

Comment:
What if more periods are added to definition? Try to make up a smarter way of defining these filter. How about storing intervals like (interval start - stop)? For today: -1day, 0 day; for last week -14days, - 7days